src/rdkafka_admin.c

```diff
@@ -2911,6 +2913,23 @@ const char *rd_kafka_ResourceType_name(rd_kafka_ResourceType_t restype) {
 }
 
 
+int8_t map_from_resource_type_to_config_resource_type(
+    rd_kafka_ResourceType_t restype) {
+        if (restype == RD_KAFKA_RESOURCE_GROUP)
```

Comment:
Let's use a `switch` to map all the options, in case we pass a `ResourceType` that cannot be used as a `ConfigResourceType` like `RD_KAFKA_RESOURCE_TRANSACTIONAL_ID` it should be mapped to `RD_KAFKA_CONFIG_RESOURCE_TYPE_UNKNOWN` not to 5 that isn't a valid `ConfigResourceType`.
Same for the reverse function.